app/fullService/api/transactionLogVersionConversion.ts

```diff
@@ -0,0 +1,137 @@
+import { MOBTOKENID } from '../../constants/app';
+import { StringB58 } from '../../types';
+import type {
+  TransactionLog,
+  TransactionLogV2,
+  Status,
+  TransactionAbbreviation,
+  TransactionLogs,
+  TransactionLogsFromV2,
+} from '../../types/TransactionLog';
+import type { InputTxo, OutputTxo } from '../../types/TxProposal';
+import type { TxosV2, TxoV2 } from '../../types/Txo.d';
+
+export function matchStatus(status: string): Status {
+  switch (status) {
+    case 'built':
+      return 'tx_status_built';
+    case 'succeeded':
+      return 'tx_status_succeded';
+    case 'pending':
+      return 'tx_status_pending';
+    case 'failed':
+      return 'tx_status_failed';
+    default:
+      return 'tx_status_succeded';
```

Comment:
default value should be `tx_status_unknown` or similar, not `tx_status_succeeded`